src/web/app/components/question-types/question-edit-answer-form/mcq-question-edit-answer-form.component.html

```diff
@@ -1,11 +1,22 @@
 <tr>
   <td>
-    <div class="radio" *ngFor="let num of questionDetails.mcqChoices; let i = index;">
-      <label class="margin-right-15px">
-        <input type="radio" class="margin-right-15px" name="mcq-{{ id }}" [checked]="isMcqOptionSelected[i]"
-               (click)="updateSelectedMcqOption(i)" [disabled]="isDisabled">
-        <span class="option-text" [innerHTML]="questionDetails.mcqChoices[i] | safeHtml"></span>
-      </label>
+    <div *ngIf="!questionDetails.questionDropdownEnabled">
+      <div class="radio" *ngFor="let num of questionDetails.mcqChoices; let i = index;">
+        <label class="margin-right-15px">
+          <input type="radio" class="margin-right-15px" name="mcq-{{ id }}" [checked]="isMcqOptionSelected[i]"
+                 (click)="updateSelectedMcqOption(i)" [disabled]="isDisabled">
+          <span class="option-text" [innerHTML]="questionDetails.mcqChoices[i] | safeHtml"></span>
+        </label>
+      </div>
+    </div>
+    <div *ngIf="questionDetails.questionDropdownEnabled" >
+      {{updateParentCss(true)}}
+      <select class="form-control margin-bottom-8px" (change)="updateSelectedMcqDropdownOption($event.target.value)" [disabled]="this.isOtherTicked">
+        <option [disabled]="!this.isOtherTicked" selected></option>
+        <option *ngFor="let num of questionDetails.mcqChoices; let i = index;"
+                [value]="questionDetails.mcqChoices[i] | safeHtml"
+                [selected]="isMcqOptionSelected[i]" [disabled]="isDisabled">{{ questionDetails.mcqChoices[i] }}</option>
+      </select>
```

Comment:
I think it would be nice to test this section as well. Perhaps some tests you can consider adding here:
- All the mcq choices in the question are displayed
- Whether the answer is updated when another dropdown option is selected

Reply:
Thanks @ypinhsuan! Will do that later today. I appreciate the suggestions!